### PR TITLE
Versioning for more models

### PR DESCRIPTION
**Pull request by marten**
A later PR will remove the papertrail gem completely, but for this one it's needed to convert the data over.

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: app/models/organization_page.rb [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
   include LanguageValidation
-  include Unversioned
-
-  has_paper_trail ignore: [:language]
+  include Versioning
 
   belongs_to :organization
+  has_many :organization_page_versions, dependent: :destroy
+
+  has_paper_trail ignore: [:language]
```
Comment (camallen) [inline app/models/organization_page.rb]: Can we remove this ? I don't think anyone is using it.

File: db/migrate/20190222121420_create_project_page_versions.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateProjectPageVersions < ActiveRecord::Migration
+  def change
+    create_table :project_page_versions do |t|
+      t.references :project_page, index: true, foreign_key: true, null: false
+      t.text :title
+      t.text :content
+      t.string :url_key
+      t.timestamps null: false
```
Comment (camallen) [inline db/migrate/20190222121420_create_project_page_versions.rb]: OOI why the null false here?

File: app/models/organization_page.rb (hunk 1 above)
Comment (marten) [reply app/models/organization_page.rb]: Yeah I'm removing all of them in a different PR.

File: db/migrate/20190222121420_create_project_page_versions.rb (hunk 2 above)
Comment (marten) [reply db/migrate/20190222121420_create_project_page_versions.rb]: IDK, Rails is now doing that.

**Comment by marten** [comment]
No model spec, but that's a reasonable idea. I'll have a go at one.

**Pull request merged**